### PR TITLE
Provide an example docker-compose.yml

### PR DESCRIPTION
**Pull request by niieani**
An easier way to use, start and restart droppy with docker-compose.

Simply copy the file and run: docker-compose up -d

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '2'
```
Comment (silverwind) [inline docker-compose.yml]: What's the significance of this version? Can I just set it to the package version?

**Comment by silverwind** [comment]
Can you move the file to the `examples` folder? I can link to it later in the README.

File: docker-compose.yml (hunk 1 above)
Comment (niieani) [inline docker-compose.yml]: No. This is the version of the docker-compose format, it has to be set to `2`. The older format was deprecated.

**Comment by niieani** [comment]
Moved.

File: docker-compose.yml (hunk 1 above)
Comment (silverwind) [inline docker-compose.yml]: Ah, alright!

**Comment by silverwind** [comment]
Thanks!

**Comment by silverwind** [comment]
And added to the README: https://github.com/silverwind/droppy#docker-installation-whale

**Pull request merged**